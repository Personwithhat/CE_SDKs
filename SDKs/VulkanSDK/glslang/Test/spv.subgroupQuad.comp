version https://git-lfs.github.com/spec/v1
oid sha256:4ebb501bc4ca9cb448c3b6f2013748f3e4406922ffaef908879d8f2253b7415a
size 6333
