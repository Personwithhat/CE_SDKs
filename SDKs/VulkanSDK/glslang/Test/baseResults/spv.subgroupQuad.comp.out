version https://git-lfs.github.com/spec/v1
oid sha256:05e4d421b49c51039f4e10c0247e1c0c66100f8e7ea774e472a5d77cc66d6d89
size 38543
